### ISSUE TITLE
Table: WCAG issues

### ISSUE THREAD

**Issue by kristiina-richards** [TEDI-Design-System/react#349]
### Description of the problem

1. When JAWS reads cell data, then it should also say the header before. ([expected to say, Age 18, not just 18](https://tehik-ee.github.io/tedi-design-system/main/iframe.html?args=&id=components-table--default&viewMode=story)) 
**Requirement:** [link](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html)
2. JAWS reads the table header sorting buttons correctly, but should also say the header name.
**Requirement:** [link](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html)
3. In filter tooltip, the search input field 'x' is read out as "Unlabeled 1, button"
**Requirement:** [link](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html)
4. Option to have the empty state accept role=alert to announce to users when table filtering doesn't have any results
